src/storage/storage_transaction.go

```diff
@@ -182,6 +182,29 @@ func (s *Storage) transactionDBIDs(txids []types.Hash32) (*[]int64, error) {
 	return &res, nil
 }
 
+func (s *Storage) TransactionsInBlock(blockId int64) (*TxIterator, error) {
+	rows, err := s.db.Query(`
+		SELECT
+			id, txid, first_seen, last_removed, fee, weight
+		FROM
+			"transaction"
+		WHERE
+			id IN (
+				SELECT
+					transaction_id
+				FROM
+					"transaction_block"
+				WHERE
+					block_id = ?
+			)`, blockId)
```

Comment:
Could benchmark if a JOIN is faster. But can be done once we run into performance issues and does not need to be done now.